WiserTaskScheduler/WiserTaskScheduler/Core/Workers/MainWorker.cs

```diff
@@ -48,6 +56,7 @@ protected override async Task ExecuteActionAsync()
         /// <inheritdoc />
         public override async Task StopAsync(CancellationToken cancellationToken)
         {
+            await slackChatService.SendChannelMessage(":skull: *Wiser Task Scheduler was terminated* :skull:");
```

Comment:
Icoontjes mogen hier weg. Zoals besproken kan deze onderaan in de functie wanneer het succesvol is afgesloten. De tekst "terminated" ook graag aanpassen naar een positievere tekst aangezien de shutdown was aangegeven en als dit onderaan staat correct is gedaan.